platform/linuxbsd/crash_handler_linuxbsd.cpp

```diff
@@ -71,10 +70,10 @@ static void handle_crash(int sig) {
 	}
 
 	// Print the engine version just before, so that people are reminded to include the version in backtrace reports.
-	if (String(VERSION_HASH).length() != 0) {
-		fprintf(stderr, "Engine version: " VERSION_FULL_NAME " (" VERSION_HASH ")\n");
+	if (!String(VERSION_HASH).is_empty()) {
+		fprintf(stderr, "Engine version: %s (%s)\n", VERSION_FULL_NAME, VERSION_HASH);
 	} else {
-		fprintf(stderr, "Engine version: " VERSION_FULL_NAME "\n");
+		fprintf(stderr, "Engine version: %s\n", VERSION_FULL_NAME);
 	}
```

Comment:
I think short `if...else` blocks like this will be more readable if the condition is not negated.